src/createRollupConfig.ts

```diff
@@ -117,15 +169,12 @@ export function createRollupConfig(
           compress: {
             keep_infinity: true,
             pure_getters: true,
-            collapse_vars: false,
+            passes: 10,
```

Comment:
does this do the same job as `collapse_vars`? i added that because it is buggy otherwise